recipe/migrations/cuda102.yaml

```
@@ -0,0 +1,26 @@
+migrator_ts: 1576782497
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    0
+
+cuda_compiler_version:         # [linux64]
+  - None                       # [linux64]
+  - 9.2                        # [linux64]
+  - 10.0                       # [linux64]
+  - 10.1                       # [linux64]
+  - 10.2                       # [linux64]
+
+docker_image:                                   # [linux]
+  - condaforge/linux-anvil-comp7                # [linux64]
+  - condaforge/linux-anvil-cuda:9.2             # [linux64]
+  - condaforge/linux-anvil-cuda:10.0            # [linux64]
+  - condaforge/linux-anvil-cuda:10.1            # [linux64]
+  - condaforge/linux-anvil-cuda:10.2            # [linux64]
+
+  - condaforge/linux-anvil-aarch64              # [aarch64]
+  - condaforge/linux-anvil-ppc64le              # [ppc64le]
+  - condaforge/linux-anvil-armv7l               # [armv7l]
```

```suggestion
  - condaforge/linux-anvil-armv7l               # [armv7l]

cuda_compiler_stub: none
```

This might work? It should trigger the bot to use this "package" as the one to build the graph from. I'm assuming that having this in the conda build config won't do anything bad.

---

No clue. Happy to try it if you think that would help 🙂